Resources/Prototypes/Entities/Clothing/OuterClothing/wintercoats.yml

```diff
@@ -0,0 +1,259 @@
+
+- type: entity
+  parent: ClothingOuterWinterCoat
+  id: ClothingOuterWinterEngi
+  name: engineering winter coat
+  description: A heavy jacket made from 'synthetic' animal furs. 
+  components:
+  - type: Sprite
+    sprite: Clothing/OuterClothing/WinterCoats/coatengi.rsi
+  - type: Clothing
+    sprite: Clothing/OuterClothing/WinterCoats/coatengi.rsi
+
+- type: entity
+  parent: ClothingOuterWinterCoat
+  id: ClothingOuterWinterGen
+  name: genetics winter coat
+  description: A heavy jacket made from 'synthetic' animal furs. 
+  components:
+  - type: Sprite
+    sprite: Clothing/OuterClothing/WinterCoats/coatgen.rsi
+  - type: Clothing
+    sprite: Clothing/OuterClothing/WinterCoats/coatgen.rsi
+
+- type: entity
+  parent: ClothingOuterWinterCoat
+  id: ClothingOuterWinterHoP
+  name: hop's winter coat 
+  description: A heavy jacket made from 'synthetic' animal furs. 
+  components:
+  - type: Sprite
+    sprite: Clothing/OuterClothing/WinterCoats/coathop.rsi
+  - type: Clothing
+    sprite: Clothing/OuterClothing/WinterCoats/coathop.rsi
+
+- type: entity
+  parent: ClothingOuterWinterCoat
+  id: ClothingOuterWinterHoS
+  name: hos's winter coat
+  description: A heavy jacket made from 'synthetic' animal furs. 
+  components:
+  - type: Sprite
+    sprite: Clothing/OuterClothing/WinterCoats/coathos.rsi
+  - type: Clothing
+    sprite: Clothing/OuterClothing/WinterCoats/coathos.rsi
+
+- type: entity
+  parent: ClothingOuterWinterCoat
+  id: ClothingOuterWinterHydro
+  name: hydroponics winter coat
+  description: A heavy jacket made from 'synthetic' animal furs. 
+  components:
+  - type: Sprite
+    sprite: Clothing/OuterClothing/WinterCoats/coathydro.rsi
+  - type: Clothing
+    sprite: Clothing/OuterClothing/WinterCoats/coathydro.rsi
+
+- type: entity
+  parent: ClothingOuterWinterCoat
+  id: ClothingOuterWinterJani
+  name: janitorial winter coat
+  description: A heavy jacket made from 'synthetic' animal furs. 
+  components:
+  - type: Sprite
+    sprite: Clothing/OuterClothing/WinterCoats/coatjani.rsi
+  - type: Clothing
+    sprite: Clothing/OuterClothing/WinterCoats/coatjani.rsi
+
+- type: entity
+  parent: ClothingOuterWinterCoat
+  id: ClothingOuterWinterMed
+  name: medical winter coat 
+  description: A heavy jacket made from 'synthetic' animal furs. 
+  components:
+  - type: Sprite
+    sprite: Clothing/OuterClothing/WinterCoats/coatmed.rsi
+  - type: Clothing
+    sprite: Clothing/OuterClothing/WinterCoats/coatmed.rsi
+
+- type: entity
+  parent: ClothingOuterWinterCoat
+  id: ClothingOuterWinterMiner
+  name: miners winter coat
+  description: A heavy jacket made from 'synthetic' animal furs. 
+  components:
+  - type: Sprite
+    sprite: Clothing/OuterClothing/WinterCoats/coatminer.rsi
+  - type: Clothing
+    sprite: Clothing/OuterClothing/WinterCoats/coatminer.rsi
+
+- type: entity
+  parent: ClothingOuterWinterCoat
+  id: ClothingOuterWinterPara
+  name: paramedics winter coat

```

Comment:
theres a mix here between ones like "medical winter coat" and "paramedic's winter coat", i'd prefer if they were all the former (besides the personal ones like "research director's winter coat")